packages/openapi/src/__tests__/actions.spec.ts

```diff
@@ -0,0 +1,140 @@
+import {
+	Configuration,
+	UserActionsApi,
+	Middleware,
+	ResponseContext,
+	ErrorContext,
+	RequestContext,
+	FetchParams,
+} from '../../client/ts'
+
+const httpLogging = false
+const runTests = false
```

Comment:
Hopefully I am missing it, but I don't see when this is set true?

Reply:
We don't currently have tests that can run without running up a whole Sofie, something to be resolved soon-ish, in the meantime we're leaving these disabled.